modules/component-store/src/component-store.ts

```diff
@@ -110,7 +113,8 @@ export class ComponentStore<T extends object> implements OnDestroy {
     return ((
       observableOrValue?: OriginType | Observable<OriginType>
     ): Subscription => {
-      let initializationError: Error | undefined;
+      let isSyncUpdate = true;
```

Comment:
I know why we have it, but could you please add comments for any future readers of this code what's the purpose for this variable.